scripts/interactions.js

```diff
@@ -308,7 +310,19 @@ damasflow_ondrop = function ( e )
 			}
 			else
 			{
-				if( confirm('Add ' + path + '?'))
+				if(oldPath===path){
+					var newWd=path.replace(/\/[^\/]*$/,"");
+					newWd=prompt("Ajouter � vos workdirs?",newWd);
```

Comment:
shouldn't the interface ask the user to add the workdir first? (before damas.search)